soda/core/tests/unit/test_telemetry.py

```diff
@@ -35,7 +35,7 @@ def mock_fn():
     dict_has_keys(span["attributes"], ["user_cookie_id"])
     dict_has_keys(span["context"], ["span_id", "trace_id", "trace_state"])
     dict_has_keys(
-        span["resource"],
+        span["resource"]["attributes"],
```

Comment:
This needs changes to our collector, I'll open another PR with all the changes and merge when the collector is updated.